### ISSUE TITLE
Mandatory fields not working in block content type

### ISSUE THREAD

**Issue by alexander-schranz** [sulu/sulu#3161]
| Q | A
| --- | ---
| Bug? | yes
| New Feature? | no
| Sulu Version | 1.4.2
| Browser Version | Tested in Chrome 55

#### Actual Behavior

If I have a required field in the xml it is not validated

#### Expected Behavior

The required field should be validated

#### Steps to Reproduce

```xml
        <block name="blocks" default-type="test">
            <types>
                <type name="test">
                    <properties>
                        <property name="test" type="text_line" mandatory="true">
                            <meta>
                                <title lang="en">Test</title>
                            </meta>
                        </property>
                    </properties>
                </type>
            </types>
        </block>
```

**Comment by nvdbeek** [comment]
Do you know when the fix is coming?

**Comment by wachterjohannes** [comment]
@nvdbeek sorry :) but there is currently no progress at least not for the 1.x release series. @danrot we should take care on this in 2.0 release development

**Comment by alexander-schranz** [comment]
fixed in #4889
